<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="greeting">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hello,</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>[USERNAME]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cashierButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Inventory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Promotions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add new Cashier </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstnameForm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="middlenameForm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastnameForm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameForm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Password: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordForm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMsg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>[INVISIBLE ERROR MESSAGE HERE]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
